Eevee.Sleep.Bot/Controllers/Mongo/MongoConst.cs

```diff
@@ -20,4 +20,12 @@ public static class MongoConst {
 
     public static readonly IMongoCollection<ActivationPresetModel> AuthActivationPresetCollection =
         AuthDatabase.GetCollection<ActivationPresetModel>("activationPreset");
+
+    private static readonly IMongoDatabase DiscordDatabase = Client.GetDatabase("role");
+
+    public static readonly IMongoCollection<RoleRecordModel> RoleRecordCollection =
```

Comment:
Prefix collection names with `Discord*`, or just make another static class so it's clear in the code that it uses the `discord` db.